docker-compose.yml

```
@@ -2,67 +2,66 @@
 ### !NOTE: This file is meant to be used by start.sh! ###
 ########################################################
 services:
-  bux-wallet-frontend:
-    build:
-      context: .
-      dockerfile: ./release/Dockerfile
+  wallet-frontend:
+    image: bsvb/spv-wallet-web-frontend:latest
+    pull_policy: always
     ports:
       - "3002:80"
     configs:
       - source: frontend-env-config
         target: /usr/share/nginx/html/env-config.json
 
-  bux-wallet-backend:
-    image: bsvb/bux-wallet-backend:latest
+  wallet-backend:
+    image: bsvb/spv-wallet-web-backend:latest
     pull_policy: always
     environment:
-      BUX_PAYMAIL_DOMAIN: '${RUN_PAYMAIL_DOMAIN}'
+      SPVWALLET_PAYMAIL_DOMAIN: '${RUN_PAYMAIL_DOMAIN}'
       HTTP_SERVER_CORS_ALLOWEDDOMAINS: 'http://localhost:3002'
     env_file:
       - .env.config
     ports:
-      - "8081:8080"
+      - "8081:8180"
```

expose it as port 8180 it will be less cumbersome
```suggestion
      - "8180:8180"
```

---

changed to 8180